edk2toollib/database/tables/instanced_fv_table.py

```diff
@@ -27,6 +28,9 @@ class InstancedFvTable(TableGenerator):
     |------------------------------------------------------|
     ```
     """  # noqa: E501
+
+    RULEOVERRIDE = re.compile(r'RuleOverride\s*=.+\s+(.*\.inf)', re.IGNORECASE)
```

Comment:
`.*\.inf` was intentionally allowing `.inf` to match, right? As opposed to `.+\.inf` which would require at least one character in the name. I think allowing only the extension to match is fine, but it could also be a mistake.

Reply:
I agree - it's more appropriate to require at least a single character before `.inf`. I will update. :)

Reply:
Updated! Thanks for the review!